api/main_endpoints/routes/URLShortener-Endpoints.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+
+let URL_SHORTENER_BASE_URL = 'http://localhost:8000';
+
+router.get('/listAll', async (req, res) => {
```

Comment:
all of these functions should require a token from the user, if you dont know how to do this lmk